test/kube_testing/pods/nsmd.go

```diff
@@ -78,9 +78,9 @@ func NSMDPod(name string, node *v1.Node) *v1.Pod {
 								Scheme: "HTTP",
 							},
 						},
-						InitialDelaySeconds: 10,
-						PeriodSeconds:       10,
-						TimeoutSeconds:      3,
+						InitialDelaySeconds: 1,
```

Comment:
Do we really need them for testing?

Reply:
That is a good question.
We might consider having a dedicated test about these and just remove them from the generic testing.

Reply:
Yes, this would be good.